packages/app/src/components/difference-indicator/tile-difference.tsx

```diff
@@ -86,8 +86,7 @@ export function TileDifference({
         <Gelijk />
       </IconContainer>
       <InlineText>
-        {text.gelijk}
-        {timespanTextNode}
+        {text.gelijk} {timespanTextNode}
```

Comment:
```suggestion
        {`${text.gelijk} ${timespanTextNode}`}
```
To make it a tad ore explicit